pkgs/servers/code-server/default.nix

```
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchFromGitHub, buildGoModule, makeWrapper, runCommand
 , moreutils, jq, git, zip, rsync, pkg-config, yarn, python2
-, nodejs-12_x, libsecret, xorg, ripgrep, nettools }:
+, nodejs-12_x, libsecret, xorg, ripgrep, darwin}:
```

Please do not input darwin directly but inherit the frameworks and packages you need.

---

Definitely, would you mind sharing an example of how to make it work? I'm pretty new to building packages on Darwin. When I did the following, then ran `nix-build ~/repos/nixpkgs -A code-server`, it errored with `error: anonymous function at /Users/derek/repos/nixpkgs/pkgs/servers/code-server/default.nix:1:1 called without required argument 'AppKit'`. 
```suggestion
, nodejs-12_x, libsecret, xorg, ripgrep, AppKit, Cocoa, Security, cctools}:
```

---

You need to add the following between the two {} in the call Package call in all-packages:

```
inherit (darwin.apple_sdk.frameworks AppKit
```

---

Never mind, I think I got it 🤦 Rebuilding to test now, then will push up an update

---

This has been updated now, thanks for the help!